.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,32 @@
+name: CodeQL
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+  schedule:
+    - cron: 27 2 * * 1,4
```

Comment:
I noticed the cron schedules are at different times in different repos. This is probably just when the PR was created, but I was wondering if there was a reason for it, or not? Also, could you add a comment about when this cron will run here please?

Reply:
I added a comment.  The times are slightly different in some of the repos because of cutting and pasting from different places.  All of the times are in the early morning of Monday and Thursday so the results will be ready when we begin our work day.

Reply:
That works. I was just wondering why they were at different times. The specific time won't really matter, we just wanted it to be early morning.